server/session.js

```diff
@@ -0,0 +1,52 @@
+const connectRedis = require('connect-redis');
+const session = require('express-session');
+const redis = require('redis');
+
+const Config = require('./config.js');
+
+const SESSION_MAX_AGE_MILLIS = 12 * 60 * 60 * 1000;
+
+const SESSION_MAX_AGE_SECONDS = SESSION_MAX_AGE_MILLIS / 1000;
```

Comment:
Trenger vi å gjøre om til sekunder for å lagre i redis, hvis vi bruker ms andre steder?

Reply:
`SESSION_MAX_AGE_MILLIS` blir brukt to steder, det ene direkte for å sette levetid for cookie og det andre indirekte for å sette levetid for Redis-session. Den siste forventer sekunder:
https://github.com/tj/connect-redis

Reply:
Det gir mening! 👍